ggshield/cmd/utils/hooks.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from ggshield.core.text_utils import display_error
+
+
+def check_user_requested_skip() -> bool:
+    if get_skip_env_var():
+        return True
+    return False
+
+
+def get_skip_env_var() -> bool:
+    """
+    When `ggshield` is configured via the [pre-commit
+    framework](https://pre-commit.com/) the user has the option to skip
+    specific scans by setting the `SKIP` environment variable. When `ggshield`
+    is configured directly as a `git` pre-commit hook, though, the user only
+    has the option to disable _all_ pre-commit hooks via the `--no-verify`
+    command-line option. Here we're "borrowing" the pre-commit framework's
+    option so ggshield can be skipped the same way regardless of how it's
+    configured.
+    """
+    try:
+        skip_env_var = os.getenv("SKIP", None)
+        if skip_env_var is not None:
+            skipped = set(p.strip().lower() for p in skip_env_var.split(","))
+            if "ggshield" in skipped:
+                return True
+    except BaseException as e:
+        display_error(f"Unable to parse SKIP environment variable: {str(e)}")
```

Comment:
I don't think we need to wrap the whole function like this: there's not really code which can fail there.

On the other hand, can you add a unit-test for this function?